lib/elixir_auth_github.ex

```diff
@@ -1,11 +1,42 @@
 defmodule ElixirAuthGithub do
   @moduledoc """
-  Documentation for ElixirAuthGithub.
+  ElixirAuthGithub is a module to help you with Github OAuth in Elixir and Phoenix.
+
+  We created it because we everyone at dwyl uses github (including our clients!) so github oauth makes sense for our internal (and external) tools. As a result, there's no use reinventing the wheel every project, and by making it into a module we can help other people as well!
```

Comment:
'because we everyone at dwyl' reads a bit strangely. Typo?

Reply:
Also oauth should consistently be OAuth

Reply:
@finnhodgkin fixed!